### ISSUE TITLE
Revise "press kit" page as "press info and contact"

### ISSUE THREAD

**Issue by SamsungGalaxyPlayer** [monero-project/monero-site#1486]
Monero does not have an obvious place for press and conference organizers to contact members of the community. I strongly recommend revise the "Press Kit" page.

First, change it to the name "Press Info and Contact" or similar

Then, place a large driving point at the top. Make the contact email extremely obvious. Add images showing Monero at conferences (I received permission from Sunita to use). At the bottom, we can continue with the current cards.

The current Monero logo card is absolutely terrible. No one ever needs "small" or "medium" sizes. Also, no one ever needs the "no background" versions. We should only provide the "white background" versions of both in large PNG and AI. Everything else wastes valuable space and causes harmful confusion.

I don't love the remaining current cards, but I'd rather leave as-is to speed things up.

### Current

![image](https://user-images.githubusercontent.com/12520755/109370751-4a648b80-7867-11eb-9fac-701e38d866a4.png)

### Proposed

![image](https://user-images.githubusercontent.com/12520755/109370734-39b41580-7867-11eb-9667-254b274f0683.png)

**Comment by xmrhaelan** [comment]
I agree this is a needed change. The proposed title is a bit long though. What about something like “Press Resources” or “For the Press”?

I think another addition that would benefit the page would be a link to https://www.getmonero.org/community/workgroups/ so reporters can see how to connect with various workgroups.

**Comment by xmrhaelan** [comment]
I agree this is a needed change. The proposed title is a bit long though. What about something like “Press Resources” or “For the Press”?

I think an additional piece to add to the page would be a link to

**Comment by erciccione** [comment]
Who is in control of press@getmonero.org?

**Comment by SamsungGalaxyPlayer** [comment]
“Be sure to include the largest privacy project in your story or event.”

Agree with this.

All "no background" and small/medium logos need to be removed though, so that will take reworking the cards a bit. I'm fine keeping the remaining at the bottom.

The screenshot text needs to be modified with the Cryptocurrency Village at Defcon this year. Use the following:

> Monero is included at various conferences such as Defcon, Grayhat, and the CCC. The Monero Konferenco, an academic conference, is hosted annually.

**Comment by erciccione** [comment]
> All "no background" and small/medium logos need to be removed though

Why so?

**Comment by erciccione** [comment]
@SamsungGalaxyPlayer I need the photo of fluffypony at defcon if we want to add it and maybe another photo of Monerokon instead of Monero Talk. I don't think would be a good idea to add references to Monero Talk, since it's run by a political candidate and we should really avoid political links.

**Comment by SamsungGalaxyPlayer** [comment]
@erciccione Neither Sunita nor Monero Talk ran for Congress. I think you're really reaching waaaaay too much.

I'll get the images over to you in a few days.

Why would someone ever want a small/medium version of the image? They can resize. Further, the no-white-background version is used improperly all the time. It's best to standardize the logo instead of having several different ones. People can always trivially make a non-white version with one click in most software if they really want it.

**Comment by erciccione** [comment]
Added the suggested changes with #1721. Note that the page still needs to be heavily updated. For now i only added the requested changes on top.

> (I received permission from Sunita to use)

Would be good to have her approval written somewhere publicly, or better, on the PR itself.
